pil_analyzer/src/pil_analyzer.rs

```diff
@@ -561,11 +593,21 @@ impl<T: FieldElement> PILContext<T> {
                     .get(name)
                     .unwrap_or_else(|| panic!("Constant {name} not found.")),
             ),
-            PolynomialReference(_) => None,
+            PolynomialReference(name) => {
+                // TODO this whole mechanism should be replaced by a generic "reference"
+                // type plus operators.
+                if !name.shift() && name.namespace().is_none() {
+                    // See if it might be a constant
+                    self.constants.get(&name.name().to_owned()).cloned()
+                } else {
+                    None
+                }
+            }
             PublicReference(_) => None,
             Number(n) => Some(*n),
             String(_) => None,
             Tuple(_) => None,
+            LambdaExpression(_, _) => todo!(),
```

Comment:
Would be good to know what needs to be done here

Reply:
Will replace by `None`.